source/common/res/features/close-reconcile-window/main.js

```diff
@@ -0,0 +1,21 @@
+(function poll() {
+  if (typeof ynabToolKit !== 'undefined' && ynabToolKit.actOnChangeInit === true) {
```

Comment:
This line is a bit different than what's in `shared/example.js`. I'll go ahead and tweak it back so it's consistent myself, but if you had a specific reason for the variance let's talk!